data/scripts/sol-flow-node-type-gen.py.in

```diff
@@ -1087,21 +1087,9 @@ def generate_description_entry(descriptions, data):
                 data_type = m["data_type"]
                 value = m["default"]
                 value = expand_value(value, data_type)
-                # TODO: should we pass thru the symbols as strings?
-                # TODO: this would solve the DBL_MIN being platform
-                # TODO: dependent and here stated INCORRECTLY as '0.0'
                 if isinstance(value, dict):
```

Comment:
Just drop this if and all is under it. This loop is useless, it was only used to convert some strings